### ISSUE TITLE
Surrounded text in query result with '[ ]' makes it invisible

### ISSUE THREAD

**Issue by burncitiesburn** [tconbeer/harlequin#647]
**Before Proceeding, please acknowledge**:
- [x] I have consulted the [Troubleshooting guide](https://harlequin.sh/docs/troubleshooting/index).
- [x] I have searched Issues and Discussions in this repo.
- [x] Feature requests should be initiated as Discussions. This is a bug report.

**Describe the bug**

If you have a field returned in a dataset which is surrounded by '[' ']' the results appear invisible.

![Screenshot 2024-09-24 at 10 02 12 AM](https://github.com/user-attachments/assets/ef7b2e9e-5a4b-4294-a60a-516cfda751b8)

**To Reproduce**

Run a query such as 
```
select '[some text]'
```

**Expected behavior**

The output should be '[some text]'

**Actual behavior**

Field displays empty.

**Contributing**
Are you interested in contributing a fix?
- [ ] Yes
- [x] Maybe
- [ ] No

**Additional context**

What is the output of `harlequin --version`?

```
harlequin, version 1.24.0

Installed Adapters:
  - mysql, version 0.2.0
  - odbc, version 0.1.1
  - duckdb, version 1.24.0
  - sqlite, version 1.24.0
  ```

What database adapter are you using with Harlequin? (Default is `duckdb`)

`odbc`

What other options are you using when invoking Harlequin? (If you are using a profile, please add relevant items from your profile here).

Can you tell us more about your system?
- Shell: zsh
- Terminal: alacritty
- OS of the shell: MacOS 14.6.1
- OS of the terminal (if different from the shell):
- Default locale / language for your OS/Terminal/Shell: en_US/utf-8

> [!TIP]
> For example, for my system, these are:
> - Bash
> - Windows Terminal
> - Ubuntu 22.04 / WSL2
> - Windows 11
> - en_US/UTF-8

How did you install Harlequin?
- [x] pipx
- [ ] conda/mamba
- [ ] pip (global install)
- [ ] pip + venv/virtualenv or pipenv
- [ ] Poetry
- [ ] uv/Rye
- [ ] other: _____________

**Comment by tconbeer** [comment]
Yeah, I know what's going on here. Thanks for the report. This is bad, will fix.
